org.raspberrypi.rpi-imager.metainfo.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!--Copyright 2020 Flathub maintainers-->
+<component type="desktop">
+  <id>org.raspberrypi.rpi-imager</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>Apache-2.0</project_license>
+  <name>Raspberry Pi Imager</name>
+  <summary>Raspberry Pi imaging utility</summary>
+  <description>
+    <p>
+      Raspberry Pi Imager downloads a .JSON file from the Raspberry Pi
+      website with a list of all current download options, ensuring you are
+      always installing the most up-to-date version.
+    </p>
+    <p>
+      Once you’ve selected an operating system from the available options,
+      the utility reads the relevant file directly from the Raspberry Pi
+      website and writes it straight to the SD card. This speeds up the
+      process quite considerably compared to the standard process of reading
+      it from the website, writing it to a file on your hard drive, and then,
+      as a separate step, reading it back from the hard drive and writing it
+      to the SD card.
+    </p>
+    <p>
+      During this process, Raspberry Pi Imager also caches the downloaded
+      operating system image – that is to say, it saves a local copy on your
+      computer, so you can program additional SD cards without having to
+      download the file again.
+    </p>
+  </description>
+  <launchable type="desktop-id">rpi-imager.desktop</launchable>
```

Comment:
```suggestion
  <launchable type="desktop-id">org.raspberrypi.rpi-imager.desktop</launchable>
```
The other desktop file won't be found

Reply:
I wrote this metainfo file with the intent to get it integrated upstream. They provide a desktop file named `rpi-imager.desktop`. How do I make this work then?

Reply:
Or perhaps the desktop file should be renamed upstream too then?

Reply:
Yes, please rename it upstream as well if possible.

Reply:
If upstream does not want it, what are the alternatives?

Reply:
Perhaps I can submit a different metainfo file upstream and use `rename-appdata-file` in the flatpak manifest?